solidity/contracts/mERC20.sol

```diff
@@ -69,4 +75,8 @@ abstract contract AbstractERC20Upgradeable is ERC20Upgradeable, ERC20BurnableUpg
     function mint(address account, uint256 amount) public onlyMinter {
         _mint(account, amount);
     }
-}
\ No newline at end of file
+
```

Comment:
Nitpicking but we could take the opportunity and clean up `setMinter` to just:
```
emit MinterChanged(minter, newMinter);
minter = newMinter;
```

Reply:
The zero-address check doesn't hurt. Also, I already did the testnet deployment in https://github.com/mezo-org/mezod/pull/467 and I'd prefer to not waste time to re-deploy everything and modify the mappings if not really needed.

Reply:
I meant the extra `address oldMinter = ` and leave the zero address check. This extra field is not needed; we can just emit the event before. But sure, it's a minor-minor and if we already deployed them, there is no need to waste time.